drivers/soundwire/bus.c

```diff
@@ -157,7 +164,8 @@ static int sdw_delete_slave(struct device *dev, void *data)
  */
 void sdw_bus_master_delete(struct sdw_bus *bus)
 {
-	device_for_each_child(bus->dev, NULL, sdw_delete_slave);
+	device_for_each_child(&bus->md->dev, NULL, sdw_delete_slave);
```

Comment:
you can still use bus->dev here, as long as it's properly initialized.

Reply:
That's a major change in this series. Now, md->dev is a child of bus->dev. That is to say bus->dev is no longer &md->dev anymore.
https://lkml.org/lkml/2020/5/11/175

Reply:
this makes absolutely no sense to me, but I am not going to argue. If this is what it takes to get the code upstream, so be it.